spec/models/answer_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+
+describe Answer do
+  let(:questionnaire) {create(:questionnaire,:id => 1)}
```

Comment:
Yeah I have used stubs and doubles everywhere except for testing sql queries. I know it is a bad practice to create test record and test queries but the prof told me to write general tests to see if db schema is up to date. Hence I had to use create for those sql tests.